src/Domain/IndexManagement/IndexConfiguration.php

```diff
@@ -17,11 +17,6 @@ private function __construct(string $name)
         $this->name = $name;
     }
 
-    public static function empty(string $name): self
```

Comment:
Why did this go?

Reply:
Unused. mutation testing detected it was unused (and untested)